influxql/ast.go

```diff
@@ -3034,6 +3040,14 @@ type NumberLiteral struct {
 // String returns a string representation of the literal.
 func (l *NumberLiteral) String() string { return strconv.FormatFloat(l.Val, 'f', 3, 64) }
```

Comment:
I didn't want to change the public name. I don't think referring to floats as numbers is very confusing since `NumberLiteral` would just reference a real number (which would be a float) rather than specifically an integer.